lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -117,6 +117,11 @@
 
 .nx-dropdown-link {
   color: var(--nx-color-link);
+  margin: 0;
+
+  &:focus {
+    border-radius: 0;
```

Comment:
Is there a reason this isn't with the rest of the focus styles up around line 87?